dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/LogTraceData.java

```diff
@@ -348,6 +418,102 @@ public String getObjectIdKey1_1() {
         return NAMES1_1[20];
     }
 
+    public String getDatetimeKeyJSON(boolean isMessageEvent) {
```

Comment:
this method (and others like it in this file and other *Data files) should be static